ui/components/app/whats-new-popup/whats-new-popup.js

```diff
@@ -41,6 +41,31 @@ function getActionFunctionById(id, history) {
   return actionFunctions[id];
 }
 
+const renderDescription = (description) => {
+  if (!Array.isArray(description)) {
+    return (
+      <div className="whats-new-popup__notification-description">
```

Comment:
[REQ] Lets render the shared piece, this div, in the callsite, and instead make this method render `<Typography variant={TYPOGRAPHY.Paragraph}>` tags to separate out the pieces.

Reply:
@brad-decker done:
<img width="418" alt="Screen Shot 2021-05-26 at 12 58 32 PM" src="https://user-images.githubusercontent.com/8732757/119723362-34dd0e80-be22-11eb-97d8-ba62ebba8a38.png">